lib/bin/node-spellchecker/build-linux-rhel.sh

```diff
@@ -1,39 +1,41 @@
 #!/bin/bash
 
-# requires node-gyp (npm install -g node-gyp) plus CLI developper tools
+# requires node-gyp (installed as a package.json dependency) plus CLI developer tools
 
-node-gyp rebuild --target=6.0.9 --arch=x64 --dist-url=https://atom.io/download/electron
+# TODO are redhat specific binaries really needed? 
```

Comment:
Is there a historic ticket or specific reason for having the linux binaries build on two separate linux distributions?  "rpm" is not really a different binary format, it's more comparable to debian's "apt" packages - the binaries inside are the same  if the distro is Linux Standard Base based.

Reply:
Purely historic, but truth be said I do not remember! It was first created in 2017, and I suppose it was not working then with unified binaries. I was also following the logic that VSCode has two separate distributions, one for rpm and another for deb world.